boards/arm/stm32h747i_disco/Kconfig.defconfig

```diff
@@ -12,6 +12,14 @@ config BOARD
 config STM32H7_DUAL_CORE
 	default y
 
+if IPM_STM32_HSEM
```

Comment:
Please also update board documentation

Reply:
OK, I will update the board documentation.